job/scriptsGen/normalizeSteps.js

```diff
@@ -73,14 +77,29 @@ function _normalizeNewFormatSteps(steps, defaultRuntime) {
       if (_.isUndefined(task.runtime.container))
         task.runtime.container = defaultIsContainer;
 
-      task.runtime.options = task.runtime.options || {};
-      if (task.runtime.container)
-        task.runtime.options =
-          _.extend(_.clone(defaultContainerOpts), task.runtime.options);
-      else
-        task.runtime.options =
-          _.extend(_.clone(defaultHostOpts), task.runtime.options);
-
+      if (task.runtime.container) {
```

Comment:
Do we want to change this? We will have to update both the default section and this if we add anything new.